database/sql/src/main/scala/cromwell/database/slick/tables/MetadataDataAccessComponent.scala

```diff
@@ -23,9 +23,9 @@ class MetadataDataAccessComponent(val driver: JdbcProfile) extends DataAccessCom
   val metadataEntriesToSummarizeQuery = {
     Compiled(
       (limit: ConstColumn[Long]) => (for {
-        (_, metadataEntry) <-
-          summaryQueueEntries join metadataEntries on (_.metadataJournalId === _.metadataEntryId)
-      } yield metadataEntry).sortBy(_.metadataEntryId).take(limit)
+        summaryEntry <- summaryQueueEntries.take(limit)
+        metadataEntry <- metadataEntries if metadataEntry.metadataEntryId === summaryEntry.metadataJournalId
```

Comment:
might this now need to make 50,000 queries to the database (one per summary queue entry) which is taken?

Reply:
No, here's the generated query:
```
select x2.`WORKFLOW_EXECUTION_UUID`, x2.`CALL_FQN`, x2.`JOB_SCATTER_INDEX`, x2.`JOB_RETRY_ATTEMPT`, x2.`METADATA_KEY`, x2.`METADATA_VALUE`, x2.`METADATA_VALUE_TYPE`, x2.`METADATA_TIMESTAMP`, x2.`METADATA_JOURNAL_ID` from (select `METADATA_JOURNAL_ID` as x3 from `SUMMARY_QUEUE_ENTRY` limit ?) x4, `METADATA_ENTRY` x2 where x2.`METADATA_JOURNAL_ID` = x4.x3 order by x2.`METADATA_JOURNAL_ID`
```

Reply:
ok - great!